faq.md

```diff
@@ -201,15 +201,29 @@ Flutter is built with C, C++, Dart, and Skia (a 2D rendering engine). See this
 
 ### How does Flutter run my code on Android? {#run-android}
 
-The engine's C/C++ code is compiled with Android's NDK, and any Dart code is AOT-compiled
-into native code. The app runs using the native instruction set (no interpreter
-is involved).
+The engine's C/C++ code is compiled with Android's NDK, and both the SDK's and
+your app's Dart code are ahead-of-time compiled into a native, ARM library. That
+library is then included in a "runner" Android project, and the whole thing's
+built into an APK. When the app is launched, it loads the Flutter library, and
```

Comment:
When launched, the app loads the Flutter library.  Any rendering, input or event handling, and so on, are delegated to the compiled Flutter and app code.